app/server/__init__.py

```diff
@@ -206,4 +207,9 @@ def encrypt_string(raw_string):
 africastalking.initialize(config.AT_USERNAME, config.AT_API_KEY)
 africastalking_client = africastalking.SMS
```

Comment:
shifted all the instantiations down to the bottom of the file, rather having them disbursed throughout function defs